assets/templates/header.html

```diff
@@ -46,7 +46,8 @@
     <li class="nav-item"><a href="forgot-password.html" class="nav-link">Forgot</a></li>
   </ul><!-- nav -->
 </section><!-- section-module -->
-    <header class="header-main header-open">
+    <header class="header-main header-open dp-header--cm">
+      <div class="dp-logo--cm"><img src="https://via.placeholder.com/180x50"></div>
```

Comment:
Why are you using this placeholder? --> "https://via.placeholder.com/180x50
I think we have one of our own in doppler.